lib/ui/text/paragraph_impl_blink.h

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_LIB_UI_TEXT_PARAGRAPH_IMPL_BLINK_H_
+#define FLUTTER_LIB_UI_TEXT_PARAGRAPH_IMPL_BLINK_H_
+
+#include "flutter/lib/ui/painting/canvas.h"
+#include "flutter/lib/ui/text/paragraph_impl.h"
+#include "flutter/lib/ui/text/text_box.h"
+#include "flutter/sky/engine/core/rendering/RenderView.h"
+#include "lib/tonic/dart_wrappable.h"
+#include "lib/txt/src/paragraph.h"
+
+namespace tonic {
+class DartLibraryNatives;
+}  // namespace tonic
+
+namespace blink {
+
+class ParagraphImplBlink : public ParagraphImpl,
+                           public ftl::RefCountedThreadSafe<ParagraphImplBlink>,
```

Comment:
Why is this object ref counted?  It seems like it should just be owned by the Paragraph object.